examples/jobs/httpd.yaml

```diff
@@ -0,0 +1,15 @@
+name: httpd
+containers:
+- image: centos/httpd
+services:
+- name: httpd
+  type: NodePort
+  ports:
+  - port: 8080
+jobs:
+- containers:
```

Comment:
yep, the containers above is for your application, and the one in jobs is for your jobs.
How do you suggest we do this? I think this correlates with the ongoing discussions in #110.